.changeset/large-stingrays-fry.md

```diff
@@ -0,0 +1,14 @@
+---
+'astro': minor
+---
+
+Add an experimental Dev Overlay. This overlay allows you to inspect your page islands, see helpful audits on performance and accessibility, and more.
+
+```ts
+// astro.config.mjs
+export default {
+  experimental: {
+    devOverlay: true
+  }
+};
+```
```

Comment:
``````suggestion

You can enable access to the dev overlay and its API by adding the following flag to your Astro config:

```ts
// astro.config.mjs
export default {
  experimental: {
    devOverlay: true
  }
};
```

Read the [Dev Overlay Plugin API documentation](https://docs.astro.build/en/reference/dev-overlay-plugin-reference/) for information about building your own plugins to integrate with Astro's dev overlay.
``````
Especially since this is the focus of the documentation, whether or not people actually start doing this now, nice to tease it here maybe?